src/coreclr/jit/gentree.cpp

```diff
@@ -17090,7 +17132,8 @@ const GenTreeLclVarCommon* GenTree::IsLocalAddrExpr() const
 //
 GenTreeLclVarCommon* GenTree::IsImplicitByrefParameterValuePreMorph(Compiler* compiler)
 {
-#if FEATURE_IMPLICIT_BYREFS && !defined(TARGET_LOONGARCH64) // TODO-LOONGARCH64-CQ: enable this.
+#if FEATURE_IMPLICIT_BYREFS && !defined(TARGET_LOONGARCH64) &&                                                         \
+    !defined(TARGET_RISCV64) // TODO-LOONGARCH64-CQ & TODO-RISCV64: enable this.
```

Comment:
Though I did not write the code, I believe this was simply missed in the LA64 bringup (LA64 predates `FEATURE_IMPLICIT_BYREFS`, so the places now under it were doing individual target `#ifdef`s).

I do not think there is a reason to pessimize RISCV here.

Reply:
This is not written by me.
It seems pushed within this
```
commit 5fda6fd23fa1d2732ca3418d5b6829dfd1e61a19
Author: Jakob Botsch Nielsen <Jakob.botsch.nielsen@gmail.com>
Date:   Mon Jan 23 17:43:26 2023 +0100

    JIT: Allow forwarding field accesses off of implicit byrefs (#80852)
    
    The JIT currently allows forwarding implicit byrefs at their last uses
    to calls, but only if the full implicit byref is used. This change
    allows the JIT to forward any such access off of an implicit byref 
    parameter.
```

Reply:
@shushanhf The original LA64 PR did not enable this function on LA64, even though it has implicit byrefs:
https://github.com/dotnet/runtime/blob/51afa067be9f0d00dac2c455ceb78e38eef1e239/src/coreclr/jit/gentree.cpp#L15872-L15884
(From the LA64 PR commit)

Since then @SingleAccretion and I refactored these functions, but the special case for LA64 remains due to the above. If this was just an oversight then can you please PR a change to enable this and `IsImplicitByrefParameterValuePostMorph` for LA64?

Reply:
OK, thanks
I will test it on LA64.

Reply:
Removed `!defined(TARGET_RISCV64)` in the PR.